pylib/cqlshlib/test/run_cqlsh.py

```diff
@@ -111,36 +103,6 @@ def timing_out_itimer(seconds):
         finally:
             signal.setitimer(signal.ITIMER_REAL, 0)
 
-@contextlib.contextmanager
-def timing_out_alarm(seconds):
```

Comment:
Why did we remove the itimer post 2.6 stuff here?

Reply:
@bowensong was this Windows specific?

Reply:
@josh-mckenzie @smiklosovic

No, the removed method is not Windows specific.

The reason for removal:
1. the only code referenced to this method was under the `if is_win(): ... else: ...` branch below, between line 129 and 143. That chunk of code is removed because they are Windows specific.
2. Python 2.6 support was ended in 2013, nearly 9 years ago. This code has been practically sitting in an unreachable branch since then.
3. the comment says it was kept for potentially faster tests, but the there's no test using it.

Any reason to keep the unused and unreachable code in the there? I can add it back if this is a blocker.

Reply:
thanks @bowensong , I think that the removal is justifiable.